src/app/core/teams/teams.service.ts

```diff
@@ -42,7 +42,10 @@ export class TeamsService {
 	create(team: Team, firstAdmin?: any): Observable<any> {
 		return this.http.put(
 			`api/team`,
-			JSON.stringify(team),
+			JSON.stringify({
+				team: team,
+				firstAdmin: null === firstAdmin ? null : firstAdmin._id || firstAdmin
```

Comment:
@abemac can we please get a test for this logic? Looking at it, I'm not sure whether the `||` applies to the full statement or just the last two parameters. Might be nice to type-check "firstAdmin" as a `string | User` too. Thanks for fixing this bug!